src/_inc/sidebar-with-footer.htm

```diff
@@ -2,13 +2,18 @@
     ——— SIDEBAR WITH FOOTER
     ===================================== -->
     <div class="layout-sidenav bg-sidebar ">
-      <div class="mobile-sticky-body-overlay"></div>
+      <!-- <div class="mobile-sticky-body-overlay"></div> -->
 
       <div id="sidebar" class="sidebar sidebar-with-footer">
         <!-- begin sidebar scrollbar -->
         <div class="app-brand">
           <a class="logo" href="index.html">
```

Comment:
Logo is not properly centered

Reply:
Ok i see this issue

Reply:
fix the brand name display on minimized sidebar